tests/swap_test.go

```diff
@@ -109,6 +109,12 @@ var _ = Describe("[Serial][sig-compute]SwapTest", Serial, decorators.SigCompute,
 			}
 			tests.UpdateKubeVirtConfigValueAndWait(kv.Spec.Configuration)
 
+			defer func() {
```

Comment:
Restoration of the KV CR is not required, there is a generic cleanup fixture that restores it to the original.

In addition, the go defer is usually not a fit in the e2e tests, as it corrupts the collected data when tests fail.

Reply:
@EdDev Hi. Do you think we can add go defer validation to the Ginkgo linter? Or this is specific to our e2e tests with regards to log collection?

Reply:
It is specific to how we collect the data on failure, not ginkgo generic.

Reply:
Dropping this commit from this PR, it can be done on a follow-up
and not mandatory to this PR

Reply:
Removed the defer and the original code because indeed we have the AfterEach